commands/moderation/tempban.js

```diff
@@ -45,18 +45,17 @@ function validTempBan(msg, args) {
     return data;
   }
 
-  const userInformation = /(<@!?\d+>)\s(\d+[yhwdms])\s(.+)$/;
+  const userInformation = /((<@!?\d+>)|(\d{17,18}))\s(\d+[yhwdms])\s(.+)$/;
```

Comment:
I wonder it it would be better here to just make the symbols optional rather than saying "or 17-18 digits":
```javascript
const userInformation = /((<@!?)?\d+>?)\s(\d+[yhwdms])\s(.+)$/
``` 

I reason this might be better is because we're not hard-coding the id limit to be 17-18 digits long.  If you look at the Discord Developer docs, you can see the Snowflake IDs are comprised in part from the number of milliseconds since 2015: https://discord.com/developers/docs/reference#snowflakes.

Knowing that, it's within the realm of possibilities that the length of the Snowflake ID could increase to 19 digits for new users in the coming years.  Not hard-coding the length would future-proof this a little more.

Reply:
Yep, good catch, thanks! I changed it to the following.

```
/((<@!?)?\d{17,}>?)\s(\d+[yhwdms])\s(.+)$/
```

So the same as yours, but just requires the snowflake to be at least 17 digits long (trying to avoid being able to enter a random string of numbers and causing the bot to crash).